orchestration/dagster_orchestration/hca_orchestration/tests/test_solids.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from unittest.mock import Mock, patch
+
+from dagster import execute_solid
+
+from hca_orchestration.solids import post_import_validate
+
+
+class SolidsTestCase(unittest.TestCase):
+    @patch("hca_utils.utils.HcaUtils.get_all_table_names")
+    @patch("hca_utils.utils.HcaUtils.get_duplicates")
+    @patch("hca_utils.utils.HcaUtils.get_file_table_names")
+    @patch("hca_utils.utils.HcaUtils.get_null_filerefs")
+    def test_post_import_validate(self, mock_all_table_names: Mock, mock_duplicates: Mock, mock_file_table_names: Mock,
+                                  mock_null_filerefs: Mock):
+        """
+        Mock bigQuery interactions and make sure the post_import_validate solid works as desired.
+        """
+
+        fake_table_names = {"fake", "names"}
+        fake_duplicate_ids = {"fake", "ids"}
+        fake_file_table_names = {"fake"}
+        fake_null_fileref_ids = {"fake", "file", "ids"}
+
+        mock_all_table_names.return_value = fake_table_names
+        mock_duplicates.return_value = fake_duplicate_ids
+        mock_file_table_names.return_value = fake_file_table_names
+        mock_null_filerefs.return_value = fake_null_fileref_ids
+
+        result = execute_solid(post_import_validate, input_values={"google_project_name": "broad-jade-dev-data",
```

Comment:
⛏️ I would change the project name and dataset_name to something very fake so there 0% chance this ever actually pokes something in dev.

Reply:
ah yes good catch, I had that there at first when I was actually hitting bq